frontend/src/components/routes/MyRoutes.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react';
+import Header from "../header/Header";
+import {Route, Routes} from "react-router-dom";
+import Profile from "../../pages/main/profile/Profile";
+import TopUp from "../../pages/main/topup/TopUp";
+import Transactions from "../../pages/main/transactions/Transactions";
+import SignIn from "../../pages/auth/signin/SignIn";
+
+const MyRoutes: React.FC = () => {
+
+    const [isLogged, setIsLogged] = useState(true);
+
+    const RouteMain = (
```

Comment:
move it and login outside of Global Routes component. They are add unnecessary rerendering if you keep it inside